### DISCUSSION TITLE
Does KAAP support multiple K8S namespaces?

### DISCUSSION THREAD

**Discussion by slawrencemd** [datastax/kaap#163] [Q&A]
Hi,

I can successfully deploy Pulsar instances via KAAP in the same K8S namespace as in which KAAP was installed, but trying to create a new pulsar cluster in a different K8S namespace creates the pulsarcluster object itself, but none of the machinery (ex: broker, zookeeper, etc) are created. No events or logs are showing anything odd - just no creation of the K8S statefulsets, deployments, pods...

What I tried was:
- create new k8s namespace
-  add the namespace field to the YAML file:
```
apiVersion: kaap.oss.datastax.com/v1beta1
kind: PulsarCluster
metadata:
  name: second-pul
  namespace: namespace-different-from-where-kaap-is
spec:
...
```
- kubectl apply -f spec.yaml 

So simply put: does this operator support deploying Pulsar clusters across multiple K8S namespaces?
